src/remoted/manager.c

```diff
@@ -207,7 +216,11 @@ void save_controlmsg(const keyentry * key, char *r_msg, size_t msg_length, int *
             w_mutex_unlock(&lastmsg_mutex);
             agent_id = atoi(key->id);
 
-            result = wdb_update_agent_connection_status(agent_id, AGENT_CS_DISCONNECTED, logr.worker_node?"syncreq":"synced", wdb_sock);
+            result = wdb_update_agent_connection_status(agent_id,
+                                                        AGENT_CS_DISCONNECTED,
+                                                        logr.worker_node?"syncreq":"synced",
+                                                        time(&disconnected_time),
```

Comment:
If the `disconnected_time` variable isn't necessary, we could use `time(NULL)` instead

Reply:
Changed